legacy/pages/faq.mdx

```diff
@@ -10,4 +10,4 @@ As it stands this is currently not possible. We always keep an eye on the limita
 1. Click on the lower left corner of your dashboard where your username is displayed. 
 2. That initates a dropdown menu. Click on `Set youself as away`.
 
-This is a method to disable your Cal.com account which won't allow any bookings once initiated. However, bookings made before turning on *away mode* will still be booked.
+This is a method to disable your Cal.com account which won't allow any bookings once initiated. However, bookings made before turning on *away mode* will still be booked.
```

Comment:
This is a weird one, there's no apparent change and yet it picks changes here. Unsure why this is, but it shouldn't be a blocker.